FirebaseFunctions/Sources/Callable+Codable.swift

```diff
@@ -162,7 +162,7 @@ public struct Callable<Request: Encodable, Response: Decodable> {
 
   // TODO: Look into handling parameter-less functions.
   @available(macOS 12.0, iOS 15.0, watchOS 8.0, tvOS 15.0, *)
-  public func stream(_ data: Request) -> AsyncThrowingStream<Response, Error> {
+  func stream(_ data: Request? = nil) -> AsyncThrowingStream<Response, Error> {
```

Comment:
At the bottom of the PR, hit the resolve conflicts button, and address the conflict in this file, choose your branch's changes and remove the base branch's changes